privacy.md

```diff
@@ -0,0 +1,111 @@
+Developer.
+
+We can only guaranty this is true when you use Jupyter Directly and not via a
+Third Party Vendor that provide hosted Jupyter Notebook, if you have questions
+about hosted Jupyter please see your vendor documentation and privacy sections.
+
+You can also look at the [JupyterLab specific privacy
+page](https://jupyterlab.readthedocs.io/en/stable/privacy_policies.html).
+
+## Does Jupyter Project **services** Collect information ?
+
+Project Jupyter run a few free services related to Jupyter, this is include but
+not limited to nbviewer, Binder, and JupyterLite. 
+
+Those services are not designed to stored any user data on Jupyter ownd
+infrastructure. Though being publicly accessible we caution user to not use
+those services to process sensitive informations.
+
+## Do Jupyter Project Forums (discourse, zulip,...) Collect information ?
+
+Jupyter is providing a few platform for users to communicate with each others, 
+this can take multiple forms, like mailing list, forums, chat... 
+
+You should not need to create an account on any on those platforms to _read_ any
+public content, and should never be required to create an account on those
+platform to _use_ jupyter software.
+
+We do not host those software, and do not fully control the data collections
+of those platforms. We advise you to read the privacy policy of those platforms,
+and contact them if you have any concern or questions about data collection. 
+
+## I a Data Privacy information form to be filled for my organisation to use Jupyter.
+
+It is likely that as part of Jupyter deployment in your company you might be
+asked to fill a form or ask a vendor to fill a form about the Data Processing
+that Project Jupyter does.
+
+This often include but is not limited to:
+
+ - which data is collected
+ - How is billing information secured
+ - Is there a background check for people having access to this information.
+ - ...
+
+Project Jupyter **is not a service provider**, **supplier** or **vendor** and so
+generally **cannot** fill out those checklist or forms, both for a **legal** and 
+often **technical** perspective.
+
+We maintain open source tools that your institution or others may install and
+operate, and it would be these entities (your it department, your vendor) that
+you contract with who are responsible for privacy of their data collection.
+
+As we do not have a business relationship with you or you institution we do not
+have customer database, we do not have billing information, and in general you
+cannot sign a contract with us.
+
+## who is my Jupyter Vendor ?
+
+There are generally two ways to use Jupyter software:
+ - In the cloud
+ - Installed on your own machines
+
+If you are using Jupyter in the cloud then look at the URL you use to access
+Jupyter, this will likely indicate who your vendor is. 
+
+If you are using Jupyter on your own machine, the way you installed Jupyter will
+tell you who your vendor is.
```

Comment:
```suggestion
## Who is my Jupyter vendor?

There are two broad ways to deploy Jupyter software:

1. Hosted by a third-party computing service provider (in the cloud)
2. Installed on your own computers on premises that your organization controls (on-premise)

If you are using Jupyter in the cloud, then look at the URL you use to access
Jupyter. This will likely indicate who your vendor is. 

If you are using Jupyter on your own machine, the way you installed Jupyter will
tell you who your vendor is. If you installed open source packages from a
freely-available repository, you probably do not have a vendor. If you installed
open source packages from a commercial vendor that provides support, contact
that vendor for support.
```